ModerationPolicy.md

```diff
@@ -0,0 +1,356 @@
+
+## Table of Contents
+
+-   [Applicability][]
+-   [Terms][]
+-   [Grounds for Moderation][]
+-   [Requesting Moderation][]
+-   [Consideration of Intent][]
+-   [Guidelines and Requirements][]
+    -   [Collaborator Posts][]
+    -   [Non-Collaborator Posts][]
+    -   [Temporary Interaction Limits][]
+    -   [Temporary and Indefinite Blocks][]
+-   [Privacy of the expressjs/moderation Repository][]
+-   [Moderation Team][]
+-   [Escalation of Issues][]
+-   [Modifications to This Policy][]
+
+## Applicability
+
+This policy applies to all repositories under these GitHub organizations: expressjs, jshttp, and pillarjs, as well as all Working Groups.
+
+Any posts, comments, issues, or pull requests that violate the [Code of Conduct][] are subject to Moderation as defined in this policy.
+
+## Terms
+
+-   _Collaborator_ refers to any individual with configured triage role or higher
+    in any expressjs GitHub organization repository. See
+    [GitHub's Repository roles documentation][] for more information.
+-   _TC_ refers to the [expressjs Technical Committee][].
+-   _Post_ refers to the content and titles of any issue, pull request, comment,
+    discussion, or wiki page.
+-   _Moderate_ means to modify, lock, or delete one or more Posts to correct or
+    address Code of Conduct violations.
+-   _Remove_ refers to the act of removing the configured write (commit)
+    permissions for an individual Collaborator's GitHub account from _all_
+    expressjs, jshttp, and pillarjs repositories as well as removing the account from
+    the expressjs, jshttp, and pillarjs organizations membership.
+-   _Block_ refers to the act of prohibiting an individual GitHub account from any
+    further participation in the expressjs, jshttp, and pillarjs organizations. A block may be
+    _temporary_ or _indefinite_.
+    -   This Moderation Policy applies only to blocking from the organization.
+        Individuals may choose to
+        [block other individuals from their personal GitHub accounts][]. This policy
+        does not restrict blocking from personal GitHub accounts.
+-   _Requester_ refers to an individual requesting Moderation on a Post.
+
+## Grounds for Moderation
+
+Any Post in violation of the expressjs [Code of Conduct][] is
+subject to Moderation.
+
+The Moderation Team is responsible for deciding what constitutes inappropriate
+behavior that may be subject to Moderation.
+
+## Requesting Moderation
+
+Anyone may request Moderation of a Post. Requesting Moderation of a Post can be
+accomplished in one of four ways:
+
+-   Via the [report@expressjs.com][] email address,
+-   Via private email to individual [Moderation Team members][],
+-   Via a new Post in the same thread as the Post being requested for Moderation,
+-   Via a new Post in the private expressjs/moderation repository.
+
+Note that Collaborators may Moderate non-Collaborator Posts at any time without
+submitting an initial request (see: [Non-Collaborator Posts][]).
```

Comment:
```suggestion
> [!NOTE] 
> Collaborators may Moderate non-Collaborator Posts at any time without
submitting an initial request (see: [Non-Collaborator Posts][]).
```

Reply:
Hi @CBID2 :wave: 

Thanks for the suggestion. The formatting change does look good. However, this would require changing the link markup from [][] to [](). 

If you could make this change, I could commit the suggestion.